widget/ui/src/theme.ts

```diff
@@ -1,6 +1,6 @@
 // TSDX + stiches doesn't work with import statement and has a bug.
 
-import type { PropertyValue } from '@stitches/react';
+import type * as Stitches from '@stitches/react';
```

Comment:
For better tree shaking, I suggest something like this:
```
import type { PropertyValue, CSS as StitchesCSS } from '@stitches/react';
```

Reply:
done